config/config.go

```diff
@@ -54,7 +54,7 @@ var DefaultConfig = Config{
 	Period: 4 * time.Hour,
 }
 
-func New(cfg *common.Config) (Config, error) {
```

Comment:
What’s the reason for changing `New` -> `NewConfig`?

Go likes it when the packagename+typename reads like this for public functions, so in usage it becomes `config.New` (vs `config.NewConfig`)

Examples:
1. https://pkg.go.dev/reflect#New
2. https://pkg.go.dev/k8s.io/client-go/kubernetes#New